lib/OpenQA/Client.pm

```diff
@@ -52,9 +52,9 @@ sub new {
             last;
         }
     }
-    # When database locking arises, the server takes some time to reply.
-    # We could also adjust sqlite_busy_timeout in DBI server side.
-    $self->inactivity_timeout(40);
+    # Scheduling a couple of hundred jobs takes quite some time - so we better don't
+    # have any timeout at all (default is 20 seconds)
+    $self->inactivity_timeout(0);
```

Comment:
Yes, infinite timeout is not good, "very high value" makes sense, e.g. 600 i.e. 10 minutes? It has to finish eventually …